### PR TITLE
Support #321: Show Nuget Package Name

### PR DESCRIPTION
**Pull request by qinezh**
@superyyrrzz @vwxyzh @chenkennt @hellosnow @ansyral

### CODE REVIEW

**Comment by qinezh** [comment]
AppVeyor build is not supported for the cli branch, so you could ignore the failed build status.

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs [hunk 1]
```diff
@@ -68,6 +73,14 @@ public override MetadataItem DefaultVisit(ISymbol symbol)
             item.Source = VisitorHelper.GetSourceDetail(symbol);
             var assemblyName = symbol.ContainingAssembly?.Name;
             item.AssemblyNameList = string.IsNullOrEmpty(assemblyName) ? null : new List<string> { assemblyName };
+#if NET451
+            var name = Path.GetFileName(ProjectPath);
+            if (name.Equals("project.json", StringComparison.OrdinalIgnoreCase)) {
+                var project = ProjectReader.GetProject(ProjectPath);
+                var packageName = $"{assemblyName}({project.Version.ToString()})";
+                item.PackageNameList = string.IsNullOrEmpty(packageName) ? null : new List<string> { packageName };
```
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs]: packageName can't be NullOrEmpty here, needn't check. checking null for ProjectPath or project or project.Version?

**Comment by vwxyzh** [comment]
👍

**Comment by superyyrrzz** [comment]
:+1:

**Pull request merged**